packages/terra-application-navigation/src/utility-menu/UtilityMenuHeaderButton.module.scss

```diff
@@ -0,0 +1,67 @@
+:local {
+  .utility-button {
+    align-items: center;
+    background-color: transparent;
+    border-width: 0;
+    color: #fff;
+    cursor: pointer;
+    display: flex;
+    font-family: inherit;
```

Comment:
We should set this to 400 or normal I believe inherit has caused some issues in odd font weight rendering in Edge.

Reply:
Updated.